index.js

```diff
@@ -1,16 +1,23 @@
+const FujiwaraError = require('./fujiwara-error');
```

Comment:
`const` declaration needs `"use strict";` mode in Node.js v4.

This module supports Node.js (>= 4).